### PR TITLE
Update url formats popular networks

### PR DESCRIPTION
**Pull request by blackdevelopa**
**Description**

Update url formats in popular network.

**Screenshots/Recordings**

_If applicable, add screenshots and/or recordings to visualize the before and after of your change_

**Issue**

Progresses [#314](https://github.com/MetaMask/mobile-planning/issues/314)

**Checklist**

* [ ] There is a related GitHub issue
* [ ] Tests are included if applicable
* [ ] Any added code is fully documented

### CODE REVIEW

File: app/util/sanitizeUrl.tsx [hunk 1]
```diff
@@ -1,3 +0,0 @@
-const sanitizeUrl = (url: string) => url.replace(/\/$/, '');
```
Comment (sethkfman) [inline app/util/sanitizeUrl.tsx:1]: Is there a reason to delete this?

File: app/components/UI/NetworkModal/NetworkDetails/index.tsx [hunk 2]
```diff
@@ -41,7 +42,7 @@ const NetworkDetails = (props: NetworkDetailsProps) => {
     },
     {
       title: strings('networks.network_rpc_url'),
-      value: rpcUrl,
+      value: hideKeyFromUrl(rpcUrl),
```
Comment (sethkfman) [inline app/components/UI/NetworkModal/NetworkDetails/index.tsx:45]: Will this apply to user added custom networks or only our pre-populated custom networks?

File: app/util/sanitizeUrl.tsx (hunk 1 above)
Comment (blackdevelopa) [reply app/util/sanitizeUrl.tsx:1]: Yes. This already exists in `sanitizaUrl.ts`

File: app/components/UI/NetworkModal/NetworkDetails/index.tsx (hunk 2 above)
Comment (blackdevelopa) [reply app/components/UI/NetworkModal/NetworkDetails/index.tsx]: Updated the PR to only pre-populated custom networks

File: app/constants/storage.ts [hunk 3]
```diff
@@ -44,3 +44,5 @@ export const WHATS_NEW_APP_VERSION_SEEN = `${prefix}WhatsNewAppVersionSeen`;
 export const REVIEW_EVENT_COUNT = 'reviewEventCount';
 
 export const REVIEW_SHOWN_TIME = 'reviewShownTime';
+
+export const themeAppearanceLight = 'light';
```
Comment (sethkfman) [inline app/constants/storage.ts:48]: Is there a reason why you are enforcing light?

Comment (blackdevelopa) [reply app/constants/storage.ts:48]: For this, it was a simple refactor outside of the core of this PR. I replace the `light` string [here](app/components/Views/Settings/NetworksSettings/NetworkSettings/index.js) with this constant

**Review by sethkfman — APPROVED** [review]
LGTM

**Review by cortisiko — APPROVED** [review]
This is 🌮 🌮 🌮

**Pull request merged**